sdk/eventgrid/azure-eventgrid/azure/eventgrid/__init__.py

```diff
@@ -5,16 +5,22 @@
 # Code generated by Microsoft (R) Python Code Generator.
 # Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
+# pylint: disable=wrong-import-position
 
-from ._patch import EventGridPublisherClient
-from ._patch import EventGridConsumerClient
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
```

Comment:
if you move this line after `from ._patch import patch_sdk as _patch_sdk` wont the first pylint no longer be needed?

Reply:
this is generated code, I think this order is to avoid needing the previous edit to do "from _patch .... blah"